### ISSUE TITLE
Add hash to insert

### ISSUE THREAD

**Issue by jamieconnor** [Appius/create-file-webpack#5]
I've got a branch to add compilation hash to the inserted file, do you want a pull request for it?

**Comment by Appius** [comment]
Yes, if you want to propose a change, please, create a pull a request. Thanks!

**Comment by luatnd** [comment]
Thanks for awesome work, I need `hash` too :)

**Comment by Appius** [comment]
I am going to make this change sometime this weekend. Thanks!

**Comment by amireldor** [comment]
Hello. Hello?

**Comment by amireldor** [comment]
I mean, I was wondering if this is implemented, I didn't see a closed/merged PR and this issue is still marked open.

**Comment by matthias-ccri** [comment]
"This weekend" never came.
